tools/analyzer_plugin/lib/src/diagnostic/duplicate_prop_cascade.dart

```diff
@@ -17,7 +17,7 @@ class DuplicatePropCascadeDiagnostic extends ComponentUsageDiagnosticContributor
     final propUsagesByName = groupBy<PropAssignment, String>(usage.cascadedProps, (prop) => prop.name.name);
     final propUsagesWithDuplicates = propUsagesByName.values.where((usages) => usages.length > 1);
     for (final propUsages in propUsagesWithDuplicates) {
-      for (var i = 0; i < propUsages.length; i++) {
+      for (var i = 0; i < propUsages.length - 1; i++) {
```

Comment:
This probably deserves a comment explaining that we're intentionally "preserving" the last instance of the dupe prop so that when the quick fix is used to remove dupes - the behavior of the component remains unchanged since the last prop of the same name always "wins".